src/base/static/components/organisms/info-modal.js

```diff
@@ -63,10 +47,22 @@ class InfoModal extends Component {
 }
```

Comment:
If there will no longer be a button that opens up this modal, then I suggest deleting InfoModalTrigger from our feedback atoms.

And if there is a button, I suggest rendering it within this component - see InviteModal and OfflineDownloadMenu for a better pattern.